.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+**related issues** #{issue1}, #{issue2}
+
+---
+
+### description
+
+{describe your PR here}
+
+### checklist
+
+- [ ] I added the related issue(s) id in the related issues section (if any)
+  - if not, delete the related issues section
+- [ ] I described my changes and my decisions in the PR description
+- [ ] I read the development guidelines of the [CONTRIBUTING.md][development-guidelines]
+- [ ] the tests pass and have been updated if relevant
+- [ ] the code quality check pass
+
+❤️ Thank you for your contribution!
```

Comment:
If is it in the PR it look like the person who create the PR thanks us for our contribution 😅

Reply:
That's also nice : )

I saw it on other repositories, I think it will be correctly understood.